src/components/Widgets/ComplianceCard/ComplianceCard.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2020 RoadieHQ
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { FC } from 'react';
+import { Box, Link, List, ListItem } from '@material-ui/core';
+import Alert from '@material-ui/lab/Alert';
+import { InfoCard, Progress, StructuredMetadataTable } from '@backstage/core';
+import { Entity } from '@backstage/catalog-model';
+import {
+  useProtectedBranches,
+  useRepoLicence,
+} from '../../../hooks/useComplianceHooks';
+import { useProjectEntity } from '../../../hooks/useProjectEntity';
+import WarningIcon from '@material-ui/icons/ErrorOutline';
+
+type ReleaseCardProps = {
+  entity: Entity;
+};
+
+const ComplianceCard: FC<ReleaseCardProps> = ({ entity }) => {
```

Comment:
PS: I would believe avoiding FC my apply here, though I understand this is Roadie code and not straight Backstage's project code, but aligning with Backstage ADRs may be preferred?

https://backstage.io/docs/architecture-decisions/adrs-adr006

(and sorry @mcalus3, I'm definitely not a TypeScript expert, learning as I go!! hope you are OK with the feedback 😄 )

Reply:
You are right, will change it in the next commit :)